sklearn/ensemble/tests/test_weight_boosting.py

```diff
@@ -74,6 +74,13 @@ def predict_proba(self, X):
     assert_array_equal(np.argmax(samme_proba, axis=1), [0, 1, 1, 1])
 
 
+def test_oneclass_proba():
+    # Test `predict_proba` robustness for one class label input.
```

Comment:
Maybe add a reference to the original issue i.e. `https://github.com/scikit-learn/scikit-learn/issues/7501`.

Nitpick: I don't think the backquotes around predict_proba are really useful.